src/Table.js

```diff
@@ -50,9 +50,9 @@ export default class Table extends React.Component {
           return {style: {"wordWrap": "break-word", "whiteSpace": "initial"}}
         }}
         showPagination={false}
-        defaultPageSize={100}
+        defaultPageSize={this.state.data.length}
```

Comment:
@sagargg what happens when there is no `data` in state yet? Maybe we should check first:

```
defaultPageSize={this.state.data && this.state.data.length}
```

Reply:
@anuveyatsu  I think we don't need that cuz empty data means an empty array. In that case, the length will be 0.

Reply:
@sagargg I'm not sure if there is a case when `this.state.data` is undefined or null. In such case it will throw an error as it doesn't have `length` method.

Reply:
@anuveyatsu  I think that should be handled somewhere else, I don't think the table will be rendered if data is undefined.

Reply:
Handled on constructor line `11`